<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>950</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>167</red>
        <green>199</green>
        <blue>231</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Soul Sync</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../model/_c5dca270-092d-4483-b12e-a031080d7df9.jpeg</normaloff>
    <normalon>../model/_c5dca270-092d-4483-b12e-a031080d7df9.jpeg</normalon>../model/_c5dca270-092d-4483-b12e-a031080d7df9.jpeg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>250</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}
</string>
    </property>
    <property name="text">
     <string>Soul Sync</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Uighur</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color: #2C3E50;
}
</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>50</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>soul_sync_logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>431</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="new_password_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_account_btn_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Delete Account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_menu_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Uighur</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{color: #2C3E50;
}
</string>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>120</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password Requirements</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="confirm_password">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>280</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>260</y>
        <width>170</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="uppercase_checkbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>At least one uppercase letter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="new_password">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Create New Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>124</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Password</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="digit_checkbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>At least one digit</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="char_length_checkbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>At least 7 characters in length</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="special_char_checkbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>At least one special character</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="lowercase_checkbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>At least one lowercase letter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirm_change_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Reset Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_password_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>420</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Uighur</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{color: #2C3E50;
}
</string>
      </property>
      <property name="text">
       <string> Delete Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>400</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Uighur</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{color: #2C3E50;
	color: rgb(255, 0, 0);
}
</string>
      </property>
      <property name="text">
       <string>⚠️ WARNING: This action is permanent!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_account_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>151</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 0, 0);
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Delete Account</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_delete_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #4682B4;
    color: white;
    font-size: 12px;
    font-weight: bold;
    border: none;
    border-radius: 8px; 
    padding: 10px 20px;
    min-width: 10px;
    transition: background-color 0.3s ease-in-out, transform 0.1s;
}
QPushButton:hover {
    background-color: #5A9BD6;
}

QPushButton:pressed {
    background-color: #36648B;
    transform: scale(0.98); 
}
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="error_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
